neurokit2/ecg/ecg_findpeaks.py

```diff
@@ -1095,3 +1126,45 @@ def _ecg_findpeaks_peakdetect(detection, sampling_rate=1000):
             NPKI = 0.125 * peak_value + 0.875 * NPKI
 
     return signal_peaks
+
+
+# Global dict mapping the method's name to its function
+all_methods = {
```

Comment:
I wonder if instead of a global dict, it wouldn't be easier and a bit shorter to have an internal function like `_ecg_findpeaks_findmethod(method)` that would do the if-elsing (which seems a bit safer than calling the output of a as one can in the future add some more steps within the block), and that we could re-call in promac. What do you think?

Reply:
This is a good idea. I will change it in a new commit.